geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionConfigImpl.java

```diff
@@ -668,6 +668,8 @@
    */
   private int threadMonitorTimeLimit = DEFAULT_THREAD_MONITOR_TIME_LIMIT;
 
+  private boolean parallelDiskStoreRecovery = DEFAULT_PARALLEL_DISK_STORE_RECOVERY;
```

Comment:
Adding it here, makes it a distribution config setup...For system property you need to add in "SystemPropertyHelper". Look into "EVICTION_SCAN_MAX_THREADS" as an example.
Also, tests to verify the system property setting are working as expected....

Reply:
This is done, including the new tests.